firmware/application/ui_navigation.cpp

```diff
@@ -136,6 +142,7 @@ void StatusTray::clear() {
         child->set_parent(nullptr);
     children_.clear();
     width_ = 0;
+    shading_ = false;
```

Comment:
Could just use something like ‘children_.size() & 1’ to alternate background color.